src/PowerLincModem.ts

```diff
@@ -870,8 +870,13 @@ export default class PowerLincModem extends EventEmitter2 {
 		return devices.filter(d => d.vendorId === '0403' && d.productId === '6001');
 	}
 
-	public static getDeviceInfo = (cat: Byte, subcat: Byte): Device | undefined =>
-		deviceDB.devices.find(d => Number(d.cat) === cat && Number(d.subcat) === subcat);
+	public static getDeviceInfo = (cat: Byte, subcat: Byte, firmware: Byte): Device | undefined => {
+		let info = deviceDB.devices.find(d => Number(d.cat) === cat && Number(d.subcat) === subcat) as Device;
+		if(info !== undefined){
+			info.firmware = `0x${firmware.toString(16).toUpperCase()}`;
```

Comment:
cat & subcat are strings here, so converting firmware to a string as well.

This static method could possibly be called by `[InsteonDevice.getDeviceInfo](https://github.com/apearson/insteon-plm/issues/40)` so that a uniform object is returned.